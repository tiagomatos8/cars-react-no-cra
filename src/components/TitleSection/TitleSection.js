import React from 'react';
import PropTypes from 'prop-types';

/**
 * TitleSection - Title section of the pages
 *
 * @param {string} title title text
 */
const TitleSection = props => {
  return (
    <section className='TitleSection-component section'>
      <div className='container'>
        <div className='columns'>
          <div className='column is-10 is-offset-1'>
            <div className='heading'>
              <h1 className='title has-text-centered has-text-success is-size-5-mobile'>
                {props.title}
              </h1>
            </div>
          </div>
        </div>
      </div>
    </section>
  );
};

TitleSection.propTypes = {
  title: PropTypes.string
};

export default TitleSection;
