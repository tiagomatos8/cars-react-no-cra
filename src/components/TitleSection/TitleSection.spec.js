import React from 'react';
import renderer from 'react-test-renderer';
import TitleSection from './TitleSection';

describe('TitleSection Component Test Suite', () => {

  it('Render TitleSection component with no props', () => {
    const component = renderer.create(<TitleSection />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
  
});