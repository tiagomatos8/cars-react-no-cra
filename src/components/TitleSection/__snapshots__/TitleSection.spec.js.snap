// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleSection Component Test Suite Render TitleSection component with no props 1`] = `
<section
  className="TitleSection-component section"
>
  <div
    className="container"
  >
    <div
      className="columns"
    >
      <div
        className="column is-10 is-offset-1"
      >
        <div
          className="heading"
        >
          <h1
            className="title has-text-centered has-text-success is-size-5-mobile"
          />
        </div>
      </div>
    </div>
  </div>
</section>
`;
